Vision/tests/Unit/V1/ImageAnnotatorClientExtensionTest.php

```diff
@@ -38,7 +39,7 @@
  * @group vision
  * @group gapic
  */
-class ImageAnnotatorClientExtensionTest extends TestCase
+class ImageAnnotatorClientExtensionTest extends GeneratedTest
```

Comment:
Should this be reverted back to `TestCase` as well?

Reply:
Good point! Done